pipelines/chat_with_website_streaming/pipeline_wrapper.py

```diff
@@ -15,18 +21,38 @@ def setup(self) -> None:
         self.pipeline = Pipeline.loads(pipeline_yaml)
 
     def run_api(self, urls: List[str], question: str) -> str:
-        log.trace(f"Running pipeline with urls: {urls} and question: {question}")
-        result = self.pipeline.run({"fetcher": {"urls": urls}, "prompt": {"query": question}})
-        return result["llm"]["replies"][0]
+        """
+        Ask a question to a list of websites.
+
+        Args:
+            urls (List[str]): List of URLs to ask the question to.
+            question (str): Question to ask the websites.
+
+        Returns:
+            str: Answer to the question.
+        """
+        result = self.pipeline.run(
+            {"fetcher": {"urls": urls}, "prompt": {"query": question}}
+        )
+
+        # result["llm"]["replies"][0] is a ChatMessage instance
+        return result["llm"]["replies"][0].text  #
 
-    def run_chat_completion(self, model: str, messages: List[dict], body: dict) -> Union[str, Generator]:
-        log.trace(f"Running pipeline with model: {model}, messages: {messages}, body: {body}")
+    def run_chat_completion(
+        self, model: str, messages: List[dict], body: dict
```

Comment:
worth updating the types? Could be more appropriate for a separate PR
```suggestion
        self, model: str, messages: list[dict], body: dict
```